engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/nLayout/area/impl/TextArea.java

```diff
@@ -28,6 +28,12 @@ public class TextArea extends AbstractArea implements ITextArea {
 
 	protected String cachedText = null;
 
+	private final char SHY_CHAR = (char) 173;
+
+	private boolean removeShy = true; // FIXME This should be configurable, depending on the emitter.
```

Comment:
Get it from a property for now?

e.g. 

    rivate boolean removeSoftHyphen = "true".equals(System.getProperty("org.eclipse.birt.softhyphen.default.??", "true"));